### PR TITLE
Add Uber Pinot blog post

### PR DESCRIPTION
**Pull request by aweisberg**

### CODE REVIEW

**Comment by aweisberg** [comment]
@haibow please review after accepting the invite to the PrestoDB org.

File: website/blog/2020-03-01-uber-pinot.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+To evaluate how well our new system works, we benchmarked query performance on Pinot data. We generated about 100 million rows in Parquet, ORC, and Pinot segments.
+
+We set up Presto and Pinot clusters on the same SSD box (32 core Intel Xeon CPU E5-2620 v4 @ 2.10GHz, 256GB memory). Then we ran Presto queries to request data from Parquet and ORC on the local disk through our Presto-Hive connector, as well as querying Pinot data through our new Presto-Pinot connector. We also queried Pinot data through the Pinot broker directly.
+
+Querying Pinot directly achieved the best query latency, as we expected. When querying through Presto, we found no significant performance differences between different data sources. We saw sporadic latency spikes when querying Parquet files, and other than that, querying Pinot had similar query latencies compared with querying Parquet and ORC files through the Hive connector.
+
+![](/img/blog/2020-03-01-uber-pinot/figure-8.png)
+
+**Figure 8.** *Query performance of querying Pinot directly vs.*
```
Comment (haibow) [inline website/blog/2020-03-01-uber-pinot.md]: vs. here is not a new line. Please merge L146-148 to a single line.

Comment (aweisberg) [reply website/blog/2020-03-01-uber-pinot.md]: Fixed

**Comment by haibow** [comment]
lgtm. Thanks for preparing the blog!

**Pull request merged**